storage/range.go

```diff
@@ -909,12 +909,20 @@ func (r *Range) maybeGossipFirstRange() error {
 	if !r.IsFirstRange() {
 		return nil
 	}
+
 	ctx := r.context()
+
+	// Gossip the cluster ID from all replicas of the first range.
+	log.Infof("gossiping first range from store %d, range %d", r.rm.StoreID(), r.Desc().RaftID)
```

Comment:
Perhaps, especially now that we can pass additional information like the range ID through a Context object.